addons/l10n_ie/models/template_ie.py

```diff
@@ -13,7 +13,6 @@ def _get_ie_template_data(self):
             'property_account_payable_id': 'l10n_ie_a9998',
             'property_account_expense_categ_id': 'l10n_ie_a9995',
             'property_account_income_categ_id': 'l10n_ie_a9996',
-            'use_anglo_saxon': False,
```

Comment:
![?](https://media.tenor.com/h1jmVbHLV6QAAAAM/vincent.gif)

Reply:
I think the `False` was not necessary. It is `None/False` by default and the option will not be activate.